statsmodels/base/tests/test_generic_methods.py

```diff
@@ -250,7 +250,7 @@ def test_zero_collinear(self):
             pytest.skip('Not completely generic yet')
 
         use_start_params = not isinstance(self.results.model,
-                                          (sm.RLM, sm.OLS, sm.WLS, sm.GLM))
+                                          (sm.RLM, sm.OLS, sm.WLS))
         self.use_start_params = use_start_params  # attach for _get_constrained
```

Comment:
Why does this need to be an attribute of `self`?  could just evaluate it again in _get_constrained?